deploy-manage/upgrade/deployment-or-cluster/upgrade-717.md

```diff
@@ -90,6 +90,15 @@ This step covers upgrading your deployment from 7.17.x to 8.19.x, following the
 
 It's highly recommended to start this upgrade from the latest 7.17.x patch release to ensure that you’re using the most recent version of the Elastic Upgrade Assistant. You should also upgrade to the latest available 8.19.x patch release so that the same benefits apply when you later upgrade to 9.x.
 
+:::::{note}
+If you use the [{{elastic-sec}} solution](/solutions/security.md), read the full [Upgrade {{elastic-sec}} from 7.17 to 8.x](https://www.elastic.co/guide/en/security/8.19/upgrade-7.17-8x.html) guide when planning the upgrade.
+
+In particular:
+* Export all custom detection rules as a backup before upgrading, in case there are issues with the detection engine after the upgrade.
+* Review [alert schema changes](https://www.elastic.co/guide/en/security/8.19/alert-schema.html) if alerts are forwarded to an external SOAR, or if you directly query alert data in custom dashboards or tools.
```

Comment:
```suggestion
* Review [alert schema changes](https://www.elastic.co/guide/en/security/8.19/alert-schema.html) if alerts are forwarded to an external SOAR, or if you directly query alert data in custom dashboards or visualizations.
```

Reply:
I explicitly wrote `tools` trying to refer to custom / external applications or "integrations" (not Elastic integrations) that might access the alerts data directly, but anyway it's not a big deal.